### PR TITLE
chore: Migrate team section to firestore

### PR DESCRIPTION
**Pull request by faraz16iqbal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,12 +14,14 @@
     "test": "jest"
   },
   "dependencies": {
+    "cross-env": "7.0.1",
```
Comment (HashirGJ8842) [inline package.json]: Why do we need this?

File: pages/team.vue [hunk 2]
```diff
@@ -170,8 +170,14 @@ export default {
     ...mapGetters('team', ['members'])
   },
   created() {
-    this.setTeamRef()
+    this.setCoreTeamRef()
+    this.setFacultyTeamRef()
   },
+  mounted() {
```
Comment (HashirGJ8842) [inline pages/team.vue]: We don't need mounted here. Once the `reference` has been set to firestore using `bindFirestoreRef`, it listens to realtime updates in `DB`. Hence, we don't need to set it again and again

File: pages/team.vue [hunk 3]
```diff
@@ -214,7 +220,8 @@ export default {
 
       return link
     },
-    ...mapActions('team', ['setTeamRef'])
+    ...mapActions('team', ['setCoreTeamRef']),
+    ...mapActions('team', ['setFacultyTeamRef'])
```
Comment (HashirGJ8842) [inline pages/team.vue]:
```suggestion
    ...mapActions('team', ['setFacultyTeamRef', 'setCoreTeamRef'])
```

File: store/team.js [hunk 4]
```diff
@@ -1,47 +1,52 @@
...
-    return bindFirebaseRef('team', teamRef)
+  setCoreTeamRef: firestoreAction(({ bindFirestoreRef }) => {
+    return bindFirestoreRef('team', coreTeamRef)
+  }),
+  setFacultyTeamRef: firestoreAction(({ bindFirestoreRef }) => {
+    return bindFirestoreRef('facultyTeam', facultyTeamRef)
   }),
 
-  setCoordinatorsRef: firebaseAction(({ getters, bindFirebaseRef }) => {
+  setCoordinatorsRef: () => {
```
Comment (HashirGJ8842) [inline store/team.js:22]: Where are we using this ?

**Review by HashirGJ8842 — COMMENTED** [review]
It's okay for now, though there should be one commit per PR

File: store/team.js (hunk 4 above)
Comment (faraz16iqbal) [reply store/team.js:22]: Home pgae?

Comment (faraz16iqbal) [reply store/team.js:22]: page

**Comment by faraz16iqbal** [comment]
I already had a PR lined up. That's why so many changes

**Comment by HashirGJ8842** [comment]
> I already had a PR lined up. That's why so many changes

Then make a PR with another branch in which only `DB` migration is taking place

**Pull request closed without merging**